scripts/o3de/o3de/android_support.py

```diff
@@ -1937,7 +1937,31 @@ def resolve_splash_overrides(self, az_android_dst_path, az_android_package_env):
         if not splash_overrides:
             return
 
-        orientation = az_android_package_env['ORIENTATION']
+        orientation_source = az_android_package_env['ANDROID_SCREEN_ORIENTATION']
+        orientation = ORIENTATION_LANDSCAPE
+
+        # Check orientation type and convert if needed
+        if isinstance(orientation_source, str):
+            if orientation_source not in ORIENTATION_MAPPING:
+                raise RuntimeError(
```

Comment:
nit: Can we use [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError) here instead of `RuntimeError`?